hack/kubearchive-install.sh

```diff
@@ -19,7 +19,8 @@ echo ${NEXT_VERSION}
 export NEXT_VERSION=${NEXT_VERSION}
 
 bash cmd/operator/generate.sh
-kubectl kustomize config/ | envsubst | ko apply -f - --base-import-paths
+kubectl kustomize config/crds | envsubst | kubectl apply -f -
+kubectl kustomize config/templates | envsubst | ko apply -f - --base-import-paths
```

Comment:
If these change I think the release process should change as well

Reply:
I firmly believe you are breaking the release process with the changes on `config/`. This is the line on the `release.sh` that uses that folder to generate the single YAML:

```
kubectl kustomize config/ | envsubst | ko resolve -f - --base-import-paths --tags=${NEXT_VERSION} > kubearchive.yaml
```

If I run that on your PR it says:

```
$ kubectl kustomize config/ | envsubst | ko resolve -f - --base-import-paths --tags=${NEXT_VERSION}
error: unable to find one of 'kustomization.yaml', 'kustomization.yml' or 'Kustomization' in directory '/home/hemartin/git/kubearchive/config'

---
```

Reply:
Yeah, but if I make the change you indicate two comments down (adding `config/kustomization.yaml` back as an aggregator, it works.

Reply:
Solved then

Reply:
@rh-hemartin, so I'm leaving this split in `hack/kubearchive-install.sh` but unchanged in `hack/release.sh`.

Reply:
Could you test to install the `kubearchive.yml` from the release command on a fresh `kind`? I think the same error may happen for users installing. I think that we may need to split the installation for release as well, I don't think you can install a CRD and a resource using it on the same file.